src/jit/gtlist.h

```diff
@@ -161,7 +161,7 @@ GTNODE(ADD_LO     , "+Lo"          ,1,GTK_BINOP)
 GTNODE(ADD_HI     , "+Hi"          ,1,GTK_BINOP)
 GTNODE(SUB_LO     , "-Lo"          ,0,GTK_BINOP)
 GTNODE(SUB_HI     , "-Hi"          ,0,GTK_BINOP)
-GTNODE(MUL_HI     , "*Hi"          ,1,GTK_BINOP)
+GTNODE(MUL_HI     , "*Hi"          ,1,GTK_BINOP) // A mul that returns the 2N bit result of an NxN multiply
```

Comment:
Ah, that makes sense. That would be another good thing to mention in the comments (I can certainly see myself, if not others, getting confused by this sometime in the future).

---
In reply to: [77713867](https://github.com/dotnet/coreclr/pull/7065#discussion_r77713867) [](ancestors = 77713867)